index.html

```diff
@@ -1,12 +1,217 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Техномарт</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Техномарт</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <h1>
+        Техномарт
+      </h1>
+      <input type="search" placeholder="Поиск:">
+      <a href="blank.html">
+        Закладки: 0
+      </a>
+      <a href="blank.html">
+        Корзина: 0
+      </a>
+      <a href="blank.html">
+        Оформить заказ
+      </a>
+      <p>Интернет-магазин<br>
+        строительных материалов<br>
+        и инструментов для ремонта</p>
+      <p>+7 (812) 555-05-55</p>
+      <p>
+        г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8
+      </p>
+      <a href="blank.html">
+        Войти
+      </a>
+      <a href="blank.html">
+        Регистрация
+      </a>
+      <ul>
+        <li><a href="index.html">Главная</a></li>
+        <li><a href="blank.html">Компания</a></li>
+        <li><a href="blank.html">Каталог</a></li>
+        <li><a href="blank.html">Спецпредложения</a></li>
+        <li><a href="blank.html">Доставка</a></li>
+        <li><a href="blank.html">Контакты</a></li>
+      </ul>
```

Comment:
Лучше убрать тег `nav` и обернуть только этот список. Тег `nav` отвечает за "глобальную" навигацию по сайту. По-этому лучше обернуть в него только ссылки что относятся ко всему сайту
```suggestion
    <nav>
      <ul>
        <li><a href="index.html">Главная</a></li>
        <li><a href="blank.html">Компания</a></li>
        <li><a href="blank.html">Каталог</a></li>
        <li><a href="blank.html">Спецпредложения</a></li>
        <li><a href="blank.html">Доставка</a></li>
        <li><a href="blank.html">Контакты</a></li>
      </ul>
     </nav>
```